cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -707,3 +708,65 @@ def _revert():
     view = navigate_to(ansible_service, "Details")
     assert verbosity[0] == view.provisioning.details.get_text_of("Verbosity")
     assert verbosity[0] == view.retirement.details.get_text_of("Verbosity")
+
+
+@pytest.mark.tier(1)
+def test_service_ansible_playbook_vault_credentials(
+    appliance,
+    request,
+    ansible_catalog_item,
+    ansible_service_catalog,
+    ansible_service_request,
+    ansible_service,
+):
+    """
+    Add vault password and test in the playbook that encrypted yml can be
+    decrypted.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        initialEstimate: 1/2h
+        tags: ansible_embed
+    """
+    creds = conf.credentials['vault_creds']['password']
+    creds_dict = {"vault_password": creds}
+    vault_creds = appliance.collections.ansible_credentials.create(
+        "Vault_Credentials_{}".format(fauxfactory.gen_alpha()), "Vault", **creds_dict
+    )
+
+    with update(ansible_catalog_item):
+        ansible_catalog_item.provisioning = {
+            "playbook": "dump_secret_variable_from_vault.yml",
+            "vault_credential": vault_creds.name,
+        }
+
+    @request.addfinalizer
+    def _revert():
+        with update(ansible_catalog_item):
+            ansible_catalog_item.provisioning = {
+                "playbook": "dump_all_variables.yml",
+                "vault_credential": "<Choose>",
+            }
+
+        service = MyService(appliance, ansible_catalog_item.name)
```

Comment:
This teardown already covers in a global fixture `ansible_service_request`

Reply:
I know it and I have added because first service request is not get deleted as `Configuration` option didn't get enable

Reply:
Removed.